sdk/deps.bzl

```diff
@@ -285,28 +285,24 @@ def daml_deps():
             patch_args = ["-p1"],
         )
 
+    # copied from https://github.com/grpc/grpc/blob/v1.67.1/bazel/grpc_deps.bzl
     if "com_google_absl" not in native.existing_rules():
         http_archive(
             name = "com_google_absl",
-            sha256 = "59d2976af9d6ecf001a81a35749a6e551a335b949d34918cfade07737b9d93c5",
-            strip_prefix = "abseil-cpp-20230802.0",
+            sha256 = "338420448b140f0dfd1a1ea3c3ce71b3bc172071f24f4d9a57d59b45037da440",
+            strip_prefix = "abseil-cpp-20240116.0",
             urls = [
-                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/abseil/abseil-cpp/archive/20230802.0.tar.gz",
-                "https://github.com/abseil/abseil-cpp/archive/20230802.0.tar.gz",
+                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/abseil/abseil-cpp/archive/20240116.0.tar.gz",
+                "https://github.com/abseil/abseil-cpp/archive/20240116.0.tar.gz",
             ],
-            patches = [
-                "@com_github_digital_asset_daml//bazel_tools:absl-mingw-win-version.patch",
-                "@com_github_digital_asset_daml//bazel_tools:absl-mingw-compiler-name.patch",
-            ],
-            patch_args = ["-p1"],
         )
 
     if "com_github_grpc_grpc" not in native.existing_rules():
         http_archive(
             name = "com_github_grpc_grpc",
-            strip_prefix = "grpc-1.60.2",
-            urls = ["https://github.com/grpc/grpc/archive/v1.60.2.tar.gz"],
-            sha256 = "8e91230f428e83d062a958e95950599e07fe60b7f13aec288215bf9423936290",
+            strip_prefix = "grpc-1.67.1",
```

Comment:
I choose 1.67 because 
- haskell_grpc says they have tested there code with 1.67.0.
- haskell_grpc does not compile with 1.70